pagefaultgames/pokerogue
# [Bug] Fixed Cosplay Pikachu and Rotom forms not being able to start with their signature moves.

## What are the changes?
This addresses issue [#2783](https://github.com/pagefaultgames/pokerogue/issues/2783)

The signature moves for the Rotom forms, and the Cosplay Pikachu forms, have been updated to now be selectable as starting moves, rather than having to be relearned using a mushroom. 

## Why am I doing these changes?
These Pokemon should be able to start with their signature moves, as they are supposed to be known by default regardless of level. 

## What did change?
Changed the level of these moves in the pokemon's learn sets from EVOLVE_MOVE (i.e. 0) to 1. Previously these were treated identically, but (apparently) the handling of level 0 vs level 1 moves was changed so that the "starting move" filter misses level 0 moves. 

### Screenshots/Videos
![image](https://github.com/pagefaultgames/pokerogue/assets/132116525/70e8d5c3-8cb4-4a32-a0a6-f722694ee311)
Previously, electric terrain would not have been selectable. 

## How to test the changes?
Go to select a Cosplay Pikachu/Rotom form as a starter, their signature moves should be selectable as a starting move. 
You will need to have these forms unlocked as starters; unfortunately, AFAIK there is currently no easy way to do this, but PR #2967 provides a potential solution. 


## Checklist
- [ x ] There is no overlap with another PR?
- [ x ] The PR is self-contained and cannot be split into smaller PRs?
- [ x ] Have I provided a clear explanation of the changes?
- [ x ] Have I tested the changes (manually)?
    - [ x ] Are all unit tests still passing? (`npm run test`)
- [ x ] Are the changes visual?
  - [ x ] Have I provided screenshots/videos of the changes?
